HelpSource/Classes/Server.schelp

```diff
@@ -631,6 +631,38 @@ Show frequency analyzer window.
 method:: meter
 Show input/output level meters.
 
+code::
+// Create a singleton ServerMeter window at the default position if no ServerMeter window is open:
+s.meter // -> a ServerMeter
+
+// Bring the ServerMeter window to the front if a ServerMeter window is already open:
+s.meter // -> a ServerMeter
```

Comment:
Just one example of s.meter is enough. The comment could clarify the behavior:
```
// Create a singleton ServerMeter, or bring the window to the front if it has already been created.
```

Reply:
Thanks, done in https://github.com/supercollider/supercollider/pull/6283/commits/4ad70d94f4ff8b4ee959c3dd6c9c60061800dc7c!